signalfx-tracing/signalfx-go-tracing/gin/README.md

```diff
@@ -0,0 +1,84 @@
+# Gin Instrumentation Example
+This is an example of producing distributed traces using the [SignalFx Tracing Library for Go](https://github.com/signalfx/signalfx-go-tracing). This example provides an instrumented API server that serves Battleship game to show some basic patterns in accessing the instrumentations of a Gin application and MongoDB drivers. You can either play by calling exposed API endpoints directly or run a simulated player.
+
+## Prerequisite
+This example requires a running MongoDB instance that the server can access. You can start a docker container by running this command.
+```sh
+$ docker run -d --name mongo -p 27017:27017 mongo
+
+```
+## Configuration
+A default set of configuration is included in [server/server.go](./server/server.go). Modify as needed before running. Note that you can choose MongoDB driver to use by changing `MongoDriver` config.
+```go
+// server.go
+
+const (
+	// ServiceName contains name of this service. This will show up on traces
+	ServiceName = "signalfx-battleship"
+	// TracingEndpoint contains a url to send traces
+	TracingEndpoint = "http://localhost:9411/api/v2/spans"
```

Comment:
Looks like we've been assuming traces go through Smart Agent or Gateway in other examples.
"http://localhost:9080/v1/trace"